ci_support/upload_or_check_non_existence.py

```diff
@@ -44,7 +44,7 @@ def upload(cli, meta, owner, channels):
     try:
         with open('binstar.token', 'w') as fh:
             fh.write(cli.token)
-        subprocess.check_call(['anaconda-client', '--quiet', '-t', 'binstar.token',
```

Comment:
P.S. Hopefully it is obvious why I couldn't put Obvious-CI into the staged-recipes repo (ah don't you love obvious recursiveness :stuck_out_tongue_winking_eye:)